bluepyentity/utils.py

```diff
@@ -82,3 +85,35 @@ def get_secret(prompt):
         stream.flush()  # issue7208
 
     return passwd
+
+
+def url_get_revision(url: str) -> bool:
+    """Get the revision number from a url or None otherwise."""
+    url = urlparse(url)
+
+    if url.query:
+        return int(url.query.replace("rev=", ""))
```

Comment:
in case there's another query element (not saying there will be but if there was) like `...?rev=1&arg=value` this would fail.

consider using `urllib.parse.parse_qs(url.query)`

Reply:
Done